xrviz/control.py

```diff
@@ -66,7 +71,7 @@ def setup_initial_values(self, initial_params={}):
 
     def set_coords(self, data):
         """
-        To set the data coords
+        To set the data coordinates.
```

Comment:
Actually this method does not set coordinates at all, right?
Instead, it is called after coordinates have been set, to update the other input panes.

It is generally better to use descriptive statements "apply data with new coordinate sets to all input panes"